.travis.yml

```diff
@@ -29,6 +61,7 @@ matrix:
       install:
         - export CC='ccache gcc-6' CXX='ccache g++-6' JOBS=2
         - ./configure
-        - make -j2 V=
+        # We already have a compile log in the above job
+        - make -j2 > /dev/null
       script:
-        - PARALLEL_ARGS='--flaky-tests=skip' make -j1 test
+        - TEST_CI_ARGS="--flaky-tests=skip --report default" make -j1 V= test-ci
```

Comment:
With the tap format used by `test-ci` Travis truncates the log in the web UI. e.g. https://travis-ci.com/nodejs/node/jobs/188985249 so you need to click through to the raw logs to see the full test results.

Reply:
Added some grepping to get it down to ~6K lines